### PR TITLE
Finish class power boxes + cloaks

### PR DESCRIPTION
**Pull request by St3eT**

### CODE REVIEW

File: L2J_DataPack/dist/game/data/stats/items/32200-32299.xml [hunk 1]
```diff
@@ -677,42 +677,82 @@
 	<item id="32264" name="Abelius' Power" additionalName="Sigel Knight" type="EtcItem">
 		<!-- Chest that contains items helpful to Sigel Knights after Awakening. -->
 		<set name="icon" val="icon.event_hero_treasure_box_i00" />
+		<set name="is_tradable" val="false" />
+		<set name="is_dropable" val="false" />
+		<set name="is_sellable" val="false" />
 		<set name="is_stackable" val="true" />
+		<set name="handler" val="ExtractableItems" />
+		<set name="capsuled_items" val="30310,1,1,100;32316,5,5,100;33502,1,1,100" />
```
Comment (jurchiks) [inline L2J_DataPack/dist/game/data/stats/items/32200-32299.xml]: This should really be reworked into something like this:
```xml
<capsuledItems>
    <item id="30310" minCount="1" maxCount="1" chance="100" />
    <item id="32316" minCount="5" maxCount="5" chance="100" />
    <item id="33502" minCount="1" /><!-- chance optional, default 100; maxCount optional, default same as minCount -->
</capsuledItems>
```
Mainly for validation purposes and simpler parsing.

Comment (UnAfraid) [inline L2J_DataPack/dist/game/data/stats/items/32200-32299.xml]: Yeah i've plans to rework not only it, but also skills, and all stuff that using this kind of format.

Comment (jurchiks) [inline L2J_DataPack/dist/game/data/stats/items/32200-32299.xml]: Cool.

Comment (Zoey76) [inline L2J_DataPack/dist/game/data/stats/items/32200-32299.xml]: We are totally agreed, we never liked this format, but we need to keep one format untill all is reparsed :smiley:

Comment (jurchiks) [inline L2J_DataPack/dist/game/data/stats/items/32200-32299.xml]: Is there something that's not parsed yet?

Comment (UnAfraid) [inline L2J_DataPack/dist/game/data/stats/items/32200-32299.xml]: That means the items won't use DocumentItem but IXMLReader

Comment (Zoey76) [inline L2J_DataPack/dist/game/data/stats/items/32200-32299.xml]: @jurchiks reparsed from current format to new format.

Comment (Zoey76) [inline L2J_DataPack/dist/game/data/stats/items/32200-32299.xml]: @UnAfraid I was checking that the last few days, I wanted to chage the tables into a better format, without being to large.

**Comment by jurchiks** [comment]
Ah, I misunderstood that.

I don't think there's a good way to turn those tables into something better AND maintain decent readability.
Why not just `<set name="effectPoint" value="fixed value" />` for single value properties and `<set name="effectPoint" values="1 2 3 4 5 6 7 8 9 10" />` for multi-value properties?

This XSD will validate it:
```xsd
<xs:element name="values" type="intValueList" />

<xs:simpleType name="intValueList">
    <xs:list itemType="xs:integer" />
</xs:simpleType>
```
And parsing it is simple.
Enforcing the length of the list to be the same as the `level` attribute is impossible anyway.

**Comment by Zoey76** [comment]
No, because different levels may have different tables.

**Comment by jurchiks** [comment]
What different levels? Enchant routes have their own `<enchantX>` element which can contain another `<mpCost values="..." .>` element without any problems.

**Comment by Zoey76** [comment]
Nevermind @jurchiks, we can check later, that list check might be a solution, since regex can't be used due it's way too heavy (I tested it already).

**Comment by jurchiks** [comment]
Well, I just hope you're not planning to overhaul ALL the XMLs at once. Do it step by step so people can learn what is switched with what. I'm sure there are lots of people out there with their own custom skills/modified existing skills etc., so if you just overhaul it all, there will be posts on forum asking what goes where.

**Comment by Zoey76** [comment]
Yeah, not sure about step by step, would lead to many commits, maybe one push with all? Anyway, let's see how it goes and who does it :dancer:

**Comment by jurchiks** [comment]
There is absolutely nothing bad about many commits. I don't get where you're coming from with this. Write meaningful commit messages so you're documenting your work step by step, and everyone can easily review it, see the progress, and suggest ideas/report bugs as you go.
Part by part only, no single push. If you do everything at once, then someone discovers it is inflexible (like Nos' drop rework), but don't want to change it, everybody loses.

**Comment by Zoey76** [comment]
I'll take it into consideration in  the future commits, thanks.

**Pull request merged**